libs/fixtures/data/electionGeneral/ui-strings.json

```diff
@@ -315,7 +315,7 @@
     "labelKeyboardSingleQuote": "'",
     "labelKeyboardSpaceBar": "space",
     "labelNumBallotContests": "Number of contests on your ballot:",
-    "labelNumBallotsScanned": "Ballots Scanned",
+    "labelNumSheetsScanned": "Sheets Scanned",
```

Comment:
I'm updating these fixtures in such a way that they are not accurate translations anymore, but they are just fixtures, and updating with new translations is too much work.